src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseBase.cpp

```diff
@@ -912,19 +915,31 @@ void CompositionalMultiphaseBase::initializeFluidState( MeshLevel & mesh,
           if( compFrac[ei][ic] < 0.0 )
             localNegativeValues += 1;
         }
+        localMinFrac.min( sumCompFrac );
+        localMaxFrac.max( sumCompFrac );
         if( LvArray::math::abs( sumCompFrac - 1.0 ) > 1e-6 )
-          localWrongSum += 1;
+          localSupFrac += 1;
+        if( LvArray::math::abs( sumCompFrac - 1.0 ) < 1e-6 )
+          localInfFrac += 1;
```

Comment:
You must adapt the computation here:
```suggestion
        if( sumCompFrac > 1.0 + 1e-6 )
          localSupFrac += 1;
        if( sumCompFrac < 1.0 - 1e-6 )
          localInfFrac += 1;
```
- use a configurable tolerance? What do you think @paveltomin ?

Reply:
fixed tolerance is probably ok